### ISSUE TITLE
New release v2.0.0-beta.3 or v2.0.0

### ISSUE THREAD

**Issue by breml** [magnusbaeck/logstash-filter-verifier#192]
@magnusbaeck I have not followed the most recent tickets, but I feel like there have not been many reports of (severe) issues since v2.0.0-beta.2 (Oct 2, 2021) so maybe we should consider releasing v2.0.0 or at least a v2.0.0-beta3 containing the latest fixes for the recent Logstash versions. What do you think?

**Comment by magnusbaeck** [comment]
We can definitely release a beta3 any time, but for 2.0.0-final I'd like to have #97 since it's a backwards incompatible change. I couldn't immediately find the branch where I've worked on it. IIRC it was more involved than expected. I can look into it towards the end of the week.

We should also upgrade the Go version and any upgradeable module in go.mod, but that's of course easy.

**Comment by breml** [comment]
@magnusbaeck #97 has landed, should we release a beta3 now for?

**Comment by magnusbaeck** [comment]
Yes! I'm off-grid until Monday night, but feel free to do it if you don't want to wait.

**Comment by breml** [comment]
No rush, I am now OOO until begin of August.

**Comment by breml** [comment]
I am back again. I suggest to get #206 and #207 merged and then release beta.3, what do you think @magnusbaeck?

**Comment by magnusbaeck** [comment]
Yes, that's exactly the idea. I was just about to push the tag when I discovered that `make release-tarballs` was broken. I'll merge and release during the weekend.

**Comment by magnusbaeck** [comment]
Beta 3 is available now.

I have an in-progress commit that shapes up README.md quite bit that I'd like to get into 2.0.0 final; see [the readme branch](https://github.com/magnusbaeck/logstash-filter-verifier/blob/readme/README.md). There's still some work to do but I think it's in decent shape.
